graphene-writer/src/main/kotlin/com/graphene/writer/store/key/handler/AbstractElasticsearchKeyStoreHandler.kt

```diff
@@ -69,9 +72,14 @@ abstract class AbstractElasticsearchKeyStoreHandler(
 
   override fun run() {
     try {
-      val metric = metrics.poll()
-      if (Objects.nonNull(metric)) {
-        addToBatch(metric)
```

Comment:
It's amazing that we were storing only one metric. 
It was a big bug. :(

Reply:
Look on the bright side, we've found it and caught that huge bug :).
Comes closer and closer to production ready version.